recipes/openvino/all/conanfile.py

```diff
@@ -100,6 +100,10 @@ def _target_x86_64(self):
     def _gna_option_available(self):
         return self.settings.os in ["Linux", "Windows"] and self._target_x86_64 and Version(self.version) < "2024.0.0"
 
+    @property
+    def _npu_option_available(self):
+        return self.settings.os in ["Linux", "Windows"] and self._target_x86_64 and Version(self.version) >= "2024.0.0"
```

Comment:
```suggestion
        return self.settings.os in ["Linux", "Windows"] and self._target_x86_64 and Version(self.version) >= "2024.1.0"
```

Reply:
Is this change related to 2024.1.0 or 2024.0.0?? 🤔

Reply:
NPU is available since 2024.1
https://github.com/openvinotoolkit/openvino/pull/23537

Reply:
@culhatsker Please, follow the ilya-lavrenov suggestion